app/styles/_overview.less

```diff
@@ -225,6 +249,7 @@
     }
     .list-pf-container {
       display: block; // so that .word-break() will work in IE
+      overflow: hidden; // so animation offsets are hidden
```

Comment:
Results in the donut tooltip being clipped.  :(

![screen shot 2017-04-17 at 9 25 00 am](https://cloud.githubusercontent.com/assets/895728/25090271/40a15eea-2350-11e7-8d0d-265f55b1d506.PNG)